### ISSUE TITLE
v13 removes support for Props declared inline

### ISSUE THREAD

**Issue by rsolomon** [brigand/babel-plugin-flow-react-proptypes#161]
I've got a bunch of files that are declared like this:

```
class StatusLabel extends React.PureComponent<{ status: Status }> {
  ...
```

After upgrading to v13, I get the following error:

```
ERROR in StatusLabel.react.js
Module build failed: TypeError: StatusLabel.react.js: Cannot read property 'Status' of undefined
    at convertToPropTypes (node_modules/babel-plugin-flow-react-proptypes/lib/convertToPropTypes.js:262:64)
    at convertToPropTypes (node_modules/babel-plugin-flow-react-proptypes/lib/convertToPropTypes.js:139:13)
    at node_modules/babel-plugin-flow-react-proptypes/lib/convertToPropTypes.js:119:20
    at Array.forEach (<anonymous>)
    at convertToPropTypes (node_modules/babel-plugin-flow-react-proptypes/lib/convertToPropTypes.js:115:21)
    at PluginPass.ClassExpressionClassDeclaration (node_modules/babel-plugin-flow-react-proptypes/lib/index.js:325:56)
    at newFn (node_modules/babel-traverse/lib/visitors.js:276:21)
    at NodePath._call (node_modules/babel-traverse/lib/path/context.js:76:18)
    at NodePath.call (node_modules/babel-traverse/lib/path/context.js:48:17)
    at NodePath.visit (node_modules/babel-traverse/lib/path/context.js:105:12)
```

However, the issue is fixed by moving the props to its own type definition:

```
type Props = { status: Status };

class StatusLabel extends React.PureComponent<Props> {
  ...
```

I'd refactor these files, but I've got ~90 of them defined this way and don't want to go through all of them right now. Any way to bring back support for inline defs?

**Comment by brigand** [comment]
Thanks for the report!

13 actually added support for that. Before 13, I think it didn't generate propTypes for that at all. Seems to be a bug. I'll look into it.

**Comment by brigand** [comment]
Should be fixed in 13.0.1. Let me know if it works

**Comment by rsolomon** [comment]
Looks like that fixed it. Thanks!
